plugins/modules/cloud/scaleway/scaleway_volume.py

```diff
@@ -123,14 +130,20 @@ def core(module):
 
     volumeByName = None
     for volume in volumes_json['volumes']:
+        if volume['project'] == project and volume['name'] == name:
+            volumeByName = volume
+
         if volume['organization'] == organization and volume['name'] == name:
```

Comment:
Does the API return both `organization` and `project` with the same value? Or only one of these two?

Reply:
I check with ansible -vvv i have 2 return from API:
organization: null
project: <ID_project>

So I guess if we pass organization has the place of project. The value of project would be null